packages/client-sdk-nodejs/src/config/transport/transport-strategy.ts

```diff
@@ -33,12 +33,24 @@ export interface TransportStrategy {
    */
   getMaxIdleMillis(): number;
 
+  /**
+   * @returns {number} the interval time in milliseconds for when each cache client should be re-initialized.
+   */
+  getMaxClientAgeMillis(): number | undefined;
+
   /**
    * Copy constructor to update the max idle connection timeout.  (See {getMaxIdleMillis}.)
    * @param {number} maxIdleMillis
    * @returns {TransportStrategy} a new TransportStrategy with the specified max idle connection timeout.
    */
   withMaxIdleMillis(maxIdleMillis: number): TransportStrategy;
+
+  /**
+   * Copy constructor to update the max age for a client.  (See {getMaxClientAgeMillis}.)
+   * @param {number} maxClientAgeMillis
+   * @returns {TransportStrategy} a new TransportStrategy with the specified max idle connection timeout.
```

Comment:
nit: says it returns strategy with new max idle connection timeout

Reply:
fixed